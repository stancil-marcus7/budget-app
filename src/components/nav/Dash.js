import React from 'react';
import ExpenseList from '../ExpenseList';
import ExpenseListFilters from '../../components/ExpenseListFilters';
import ExpenseDashboardPage from '../pages/ExpenseDashboardPage';
const Dash = () => (
  <div>
      <ExpenseDashboardPage/>
      <ExpenseListFilters/>
      <ExpenseList/>
  </div>
)

export default Dash;
