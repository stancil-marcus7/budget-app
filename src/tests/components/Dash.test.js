import React from 'react';
import { shallow } from 'enzyme';
import Dash from '../../components/nav/Dash';

test(`should render the dash correctly`, () => {
    const wrapper = shallow(<Dash/>);
    expect(wrapper).toMatchSnapshot();
});