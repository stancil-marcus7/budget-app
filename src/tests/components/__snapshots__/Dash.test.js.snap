// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the dash correctly 1`] = `
<div>
  <Connect(ExpenseDashboardPage) />
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
